core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/ambari/AmbariLdapService.java

```diff
@@ -58,6 +59,9 @@ public void setupLdap(Stack stack, Cluster cluster, AmbariRepo ambariRepo) {
             ldapConfigs.put("ambari.ldap.connectivity.anonymous_bind", false);
             if (ambariRepositoryVersionService.isVersionNewerOrEqualThanLimited(ambariRepo::getVersion, AMBARI_VERSION_2_7_2_0)) {
                 ldapConfigs.put("ambari.ldap.advanced.collision_behavior", "convert");
+                if (ambariRepositoryVersionService.isVersionNewerOrEqualThanLimited(ambariRepo::getVersion, AMBARI_VERSION_2_7_100_0)) {
+                    ldapConfigs.put("ambari.ldap.advanced.disable_endpoint_identification", true);
+                }
             } else {
                 ldapConfigs.put("ambari.ldap.advance.collision_behavior", "convert");
```

Comment:
this should be a skip as well @keyki

Reply:
This is set in such way because it does not work otherwise (https://hortonworks.jira.com/browse/BUG-113369?focusedCommentId=891513&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-891513)